strapi/strapi
# [Audit logs] Emit user events

### What does it do?

- Emits audit log events for CRUD operations triggered in the admin `/admin/settings/users`

### How to test it?

- Go to `/admin/settings/users`
- Click invite user, when the user is created you should see the log from the audit logs service for `user.create`
- From `/admin/settings/users/{id}`, edit something. you should see the log from the audit logs service for `user.update`
- Go back to `/admin/settings/users`, delete your created user. You should see the log from the audit logs service for `user.delete`

Verify the payload is sanitized correctly and no sensitive data is logged
